src/ScreenResource.cc

```diff
@@ -109,6 +107,8 @@ ScreenResource::ScreenResource(FbTk::ResourceManager& rm,
     menu_delay(rm, 200, scrname + ".menuDelay", altscrname+".MenuDelay"),
     tab_width(rm, 64, scrname + ".tab.width", altscrname+".Tab.Width"),
     tooltip_delay(rm, 500, scrname + ".tooltipDelay", altscrname+".TooltipDelay"),
+    workspace_warping_horizontal_offset(rm, 1, scrname+".workspacewarpinghorizontaloffset", altscrname+".WorkspaceWarpingHorizontalOffset"),
```

Comment:
merge artifact?

Reply:
Perhaps; didn't come up this time around.